.eslintrc.yml

```diff
@@ -9,7 +9,7 @@ extends:
     - "plugin:eslint-plugin/recommended"
 rules:
     eslint-plugin/consistent-output: "error"
-    eslint-plugin/no-identical-tests: "error"
```

Comment:
It's redundant to have `eslint-plugin/no-identical-tests` here because it's part of the `plugin:eslint-plugin/recommended` config.